addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -302,6 +298,30 @@
                                          'invisible': [('mail_server_available', '=', False)]}"/>
                                         <field name="keep_archives" attrs="{'readonly': [('state', 'in', ('sending', 'done'))]}"/>
                                     </group>
+                                    <group string="A/B Testing" attrs="{'invisible': [('testing_mailing_id', '=', False)]}">
+                                        <field name="testing_mailing_id" readonly="1" attrs="{'invisible': [('testing_mailing_id', '=', False)]}"/>
+                                        <label for="contact_ab_pc" string="Sample Size" attrs="{'invisible': [('testing_mailing_id', '=', False)]}"/>
+                                        <span attrs="{'invisible': [('testing_mailing_id', '=', False)]}">
+                                            <field name="contact_ab_pc" readonly="1" widget="percentage" nolabel="1"/>
+                                            <span attrs="{'invisible': [('total', '=', 0)]}">
+                                                (<field name="total" class="mr-1"/><field name="mailing_model_real_display_name" readonly="1"/>)
+                                            </span>
+                                        </span>
+                                        <field name="version_id" options="{'no_open': True}" attrs="{'invisible': [('testing_mailing_id', '=', False)]}"/>
+                                        <field name="unique_ab_testing"
```

Comment:
This directly conflicts with the invisible setting on the group.
Do we still need this "unique_ab_testing" mechanic? Maybe something to ask Luc?